src/presence.rs

```diff
@@ -168,6 +183,21 @@ fn on_presence_updated(
                     activity.end = Some(current_time + ig_end);
                 }
             }
+            // This will override previous details established whenever server is not in the Playing gamestate, so friends can see at which stage a match currently is
+            if cl_presence.servergamestate != 4 {
+                activity.details = match cl_presence.servergamestate {
+                    1 => "Waiting Players to Load",
+                    2 => "Titan Selection",
+                    3 => "Match Starting",
+                    5 => "In Sudden Death",
+                    6 => "Switching Sides",
+                    7 => "Winner Determined",
+                    8 => "In Epilogue",
+                    9 => "Match Ending",
+                    _ => "",
+                }
+                .to_string();
+            }
```

Comment:
this should use a `enum` instead of just some numbers

Reply:
Literally check last commit, i really tried to enum it, but idk how to do it without rust asking me to mirror it on squirrel side which is pointless because there's already Respawn's one in there.

Reply:
Alr changed buncha stuff at the request.